cfn/prod-release.yml

```diff
@@ -176,14 +173,14 @@ Resources:
     Type: "AWS::SSM::Parameter"
     Properties:
       Description: Parameter to store our account id so CodeBuild specs can access it
-      Name: /CodeBuild/AccountId
+      Name: /CodeBuild/Account
```

Comment:
These need to match this:
https://github.com/aws/aws-encryption-sdk-java/blob/ad9a05c59a5c8f280da8bbd49de7d4f8b2232121/codebuild/release/validate-staging.yml#L9

which is `/CodeBuild/AccountId`

Reply:
That's a little weird, shouldn't this have been caught in the stack creation process? or would this be a case where it would detect a drift. I'll go ahead and change it so they match. The name was causing it to not create the stack, and changing it to `Account` successfully created the stack.

Reply:
Ohh... then nevermind. The name already exists, right? Because we have been using it.